tutorials/export/exporting_for_android.rst

```diff
@@ -137,23 +140,16 @@ Don't forget to uncheck the **Export With Debug** checkbox while exporting.
 
 .. image:: img/export-with-debug-button.png
 
-Optimizing the APK size
------------------------
+Optimizing the file size
+------------------------
 
-By default, the APK will contain native libraries for both ARMv7 and ARMv8
-architectures. This increases its size significantly. To create a smaller APK,
+By default, the AAB or APK will contain native libraries for both ARMv7 and ARMv8
+architectures. This increases its size significantly. To create a smaller file,
 uncheck either **Armeabi-v 7a** or **Arm 64 -v 8a** in your project's Android
-export preset. This will create an APK that only contains a library for
+export preset. This will create an file that only contains a library for
 a single architecture. Note that applications targeting ARMv7 can also run on
 ARMv8 devices, but the opposite is not true.
```

Comment:
With AAB, the practice of including less architectures to reduce download size is obsolete. Google automatically splits up the AAB on their backend so the user only downloads the architecture they need.